### PR TITLE
docs(cce): fix docs issues.

### PR DESCRIPTION
**Pull request by wangzepeng666**
fix docs issues.

**data_source_flexibleengine_cce_clusters**
1、add attribute field
image
image

### CODE REVIEW

File: docs/resources/cce_cluster_v3.md [hunk 1]
```diff
@@ -63,47 +67,50 @@ The following arguments are supported:
...
 
-* `cluster_type` - (Required) Cluster Type, possible values are VirtualMachine and BareMetal.
+* `cluster_type` - (Required, String, ForceNew) Cluster Type, possible values are VirtualMachine and BareMetal.
   Changing this parameter will create a new cluster resource.
 
-* `description` - (Optional) Cluster description.
+* `description` - (Optional, String) Cluster description.
 
-* `billing_mode` - (Optional) Charging mode of the cluster, which is 0 (on demand).
+* `billing_mode` - (Optional, Int, ForceNew) Charging mode of the cluster, which is 0 (on demand).
```
Comment (chengxiangdong) [inline docs/resources/cce_cluster_v3.md:79]: FE does not require change mode arg.

File: docs/data-sources/cce_clusters.md
[the remark this answers is not in the document]
Comment (wangzepeng666) [reply docs/data-sources/cce_clusters.md:1]: Referencing Huawei Cloud, not processed temporarily

File: docs/data-sources/cce_node_v3.md
Comment (chengxiangdong) [inline docs/data-sources/cce_node_v3.md:1]: The validity of the arguments needs to be tested.

File: docs/resources/cce_node_pool_v3.md
[the remark this answers is not in the document]
Comment (wangzepeng666) [reply docs/resources/cce_node_pool_v3.md:1]: `scall_enable`  is deprecated
`extend_param`  declared in schema only, but it is not used

File: docs/resources/cce_addon_v3.md
Comment (chengxiangdong) [inline docs/resources/cce_addon_v3.md:1]: ditto

File: docs/resources/cce_node_v3.md
Comment (chengxiangdong) [inline docs/resources/cce_node_v3.md:1]: ditto

**Review by chengxiangdong — CHANGES_REQUESTED** [review]

File: docs/resources/cce_cluster_v3.md (hunk 1 above)
Comment (wangzepeng666) [reply docs/resources/cce_cluster_v3.md:79]: the schema and docs already existes

File: docs/data-sources/cce_node_v3.md
Comment (wangzepeng666) [reply docs/data-sources/cce_node_v3.md:1]: unit test has but docs miss the field

Comment (wangzepeng666) [reply docs/data-sources/cce_node_v3.md:1]: unit test has but docs miss the field

File: docs/resources/cce_addon_v3.md
Comment (wangzepeng666) [reply docs/resources/cce_addon_v3.md:1]: ditto

File: docs/resources/cce_node_v3.md
Comment (wangzepeng666) [reply docs/resources/cce_node_v3.md:1]: unit test has but docs miss the field

File: docs/resources/cce_addon_v3.md
Comment (wangzepeng666) [reply docs/resources/cce_addon_v3.md:1]: this docs have no field which is new added

File: docs/data-sources/cce_clusters.md
Comment (chengxiangdong) [inline docs/data-sources/cce_clusters.md:1]: `clusters.highway_subnet_id` does not exist in schema

File: docs/resources/cce_node_v3.md [hunk 2]
```diff
@@ -40,63 +40,71 @@ resource "flexibleengine_cce_node_v3" "node_1" {
...
 
-* `cluster_id` - (Required) ID of the cluster. Changing this parameter will create a new resource.
+* `region` - (Optional, String, ForceNew) Specifies the region in which to create the CCE node resource.
+  If omitted, the provider-level region will be used. Changing this will create a new CCE node resource.
 
-* `name` - (Optional) Node Name.
+* `cluster_id` - (Required, String, ForceNew) ID of the cluster. Changing this parameter will create a new resource.
 
-* `flavor_id` - (Required) Specifies the flavor id. Changing this parameter will create a new resource.
+* `name` - (Optional, String, ForceNew) Node Name. Changing this parameter will create a new resource.
```
Comment (chengxiangdong) [inline docs/resources/cce_node_v3.md:48]:
```suggestion
* `name` - (Optional, String) Node Name.
```

Comment (wangzepeng666) [reply docs/resources/cce_node_v3.md:48]: you have see wrong, name is forceNew

File: docs/data-sources/cce_cluster_v3.md [hunk 3]
```diff
@@ -21,15 +21,20 @@ data "flexibleengine_cce_cluster_v3" "cluster" {
 
 The following arguments are supported:
 
-* `name` -  (Optional)The Name of the cluster resource.
+* `region` - (Optional, String) The region in which to query the data source. If omitted, the provider-level region
+  will be used.
 
-* `id` - (Optional) The ID of container cluster.
+* `name` -  (Optional, String)The Name of the cluster resource.
```
Comment (ShiChangkuo) [inline docs/data-sources/cce_cluster_v3.md:27]:
```suggestion
* `name` - (Optional, String)The Name of the cluster resource.
```

**Pull request merged**